client/components/thank-you/types.ts

```diff
@@ -65,4 +65,6 @@ export type ThankYouData = [
 	React.ReactElement | null,
 ];
 
+export type ThankYouThemeData = [ any[], ...ThankYouData ];
```

Comment:
Are you trying to hack the type system to avoid typing? 🤣 

We have an eslint rule that forbids the use of `any`. The only problem I've seen is that the Theme type is missing some props. 

You can find the theme type here `import { Theme } from "calypso/types"` and we should add the `style_variations` definition. I think this would do (but I haven't tested it):

```
diff --git a/client/components/thank-you/types.ts b/client/components/thank-you/types.ts
index d6b6f52a0a..aae5b4c758 100644
--- a/client/components/thank-you/types.ts
+++ b/client/components/thank-you/types.ts
@@ -1,5 +1,6 @@
 import { TranslateResult } from 'i18n-calypso';
 import * as React from 'react';
+import { Theme } from 'calypso/types';
 
 export type ThankYouNextStepProps = {
        stepCta?: React.ReactNode;
@@ -65,6 +66,6 @@ export type ThankYouData = [
        React.ReactElement | null,
 ];
 
-export type ThankYouThemeData = [ any[], ...ThankYouData ];
+export type ThankYouThemeData = [ Theme[], ...ThankYouData ];
 
 export type ThankYouSteps = { steps: string[]; additionalSteps: string[] };
diff --git a/client/types.ts b/client/types.ts
index 0d207ac483..148dcff9bb 100644
--- a/client/types.ts
+++ b/client/types.ts
@@ -42,6 +42,9 @@ export interface Theme {
        screenshot: string;
        screenshots: string[];
        stylesheet: string;
+       style_variations: {
+               slug: string;
+       }[];
        supportDocumentation: string;
        tags: string[];
        taxonomies?: {
(END)

```

Reply:
Yeah... :D I wanted to use a type but noticed that `Theme` didn't have a `style_variations` property. I guess it's fine to add it.